src/components/share-overlay/share-overlay.js

```diff
@@ -509,8 +510,10 @@ class ShareOverlay extends Component {
    * @memberof ShareOverlay
    */
   _addKalturaClipParams(url: string): string {
-    url = this._updateUrlParams(url, 'kalturaSeekFrom', this.state.clipStartTimeValue);
-    return this._updateUrlParams(url, 'kalturaClipTo', this.state.clipEndTimeValue);
+    const seekTime = Math.floor(this._convertTimeValue(this.state.clipStartTimeValue) / 2) * 2;
```

Comment:
since the calculation is repeated in different places, i would suggest to extract it to a new function.
this way, if there will be any issue with the calculation or we'd want to change it, we'll have exactly 1 place to change; also, the name of the new function can give us more information about the calculation purpose.

Reply:
Moved to cropTimeForFrames function